src/environments/environment.ts

```diff
@@ -3,7 +3,14 @@
 // The list of file replacements can be found in `angular.json`.
 
 export const environment = {
-  production: false
+  production: false,
+  version: "/v1.0",
+  applicationName: "/ORP"
+};
+
+export const loginsetting = {
```

Comment:
loginsetting should be part of environment
eg.
export const environment = {
loginsetting:'',
abc:'',
xyz:''
}

This will help us to store all constant under single variable i.e. environment

Reply:
Okay, Done